### ISSUE TITLE
HTTP POST to GitHub with a too big body fails

### ISSUE THREAD

**Issue by karlcow** [webcompat/webcompat.com#2717]
We need to check and handle the fact that a body which is too large will fail.
The body limit for posting an issue on GitHub is 65536 characters. Anything bigger will fail.

**Comment by miketaylr** [comment]
> Anything bigger will fail.

I wonder if we can handle this on the backend... detect if it's too big... and post a follow-up comment if so (very hack-y). Or, post relevant details to some other text/paste service we own and just link to that instead of putting inside the body.

**Comment by karlcow** [comment]
yup see the agenda of Orlando Moz All Hands for crazy, hacky, silly ideas. ;)
https://wiki.mozilla.org/Compatibility/Meetings/2018-12-work-week#Improve_the_console.log_reporting_by_the_Reporter_extension
